src/debuginfo/line_info.rs

```diff
@@ -57,13 +73,17 @@ fn line_program_add_file(
                 line_program.encoding(),
                 line_strings,
             );
-            line_program.add_file(file_name, dir_id, None)
+
+            let info = make_file_info(file.src_hash);
+
+            line_program.file_has_md5 = info.is_some();
```

Comment:
```suggestion
            line_program.file_has_md5 &= info.is_some();
```

According to https://github.com/bjorn3/rustc_codegen_cranelift/pull/972#discussion_r410781643 it is either every file has an MD5 or none.

Reply:
Thanks, sorry for a delay